packages/zoe/src/contracts/mintPayments.js

```diff
@@ -1,81 +1,63 @@
  *
- * makeInstance returns an invitation that, when exercised, provides 1000 of the
- * new tokens. await E(publicAPI).makeInvite() returns an invitation that accepts an
- * empty offer and provides 1000 tokens.
+ * To pay others in tokens, the creator of the instance can make
+ * invitations for them, which when used to make an offer, will payout
+ * the specified amount of tokens.
  *
- * @param {ContractFacet} zcf
+ * @type {ContractStartFn}
  */
-const makeContract = zcf => {
-  // Create the internal token mint for a fungible digital asset
-  const { issuer, mint, amountMath } = makeIssuerKit('tokens');
-
-  const zoeHelpers = makeZoeHelpers(zcf);
-
-  // We need to tell Zoe about this issuer and add a keyword for the
-  // issuer. Let's call this the 'Token' issuer.
-  return zcf.addNewIssuer(issuer, 'Token').then(() => {
-    // We need to wait for the promise to resolve (meaning that Zoe
-    // has done the work of adding a new issuer).
-    const offerHook = offerHandle => {
-      // We will send everyone who makes an offer 1000 tokens
-
-      const tokens1000 = amountMath.make(1000);
-      const payment = mint.mintPayment(tokens1000);
-
-      // Let's use a helper function which escrows the payment with
-      // Zoe, and reallocates to the recipientHandle.
-      return zoeHelpers
-        .escrowAndAllocateTo({
-          amount: tokens1000,
-          payment,
-          keyword: 'Token',
-          recipientHandle: offerHandle,
-        })
-        .then(() => {
-          // Complete the user's offer so that the user gets a payout
-          zcf.complete(harden([offerHandle]));
-
-          // Since the user is getting the payout through Zoe, we can
-          // return anything here. Let's return some helpful instructions.
-          return 'Offer completed. You should receive a payment from Zoe';
-        });
-    };
-
-    // A function for making invites to this contract
-    const makeInvite = () => zcf.makeInvitation(offerHook, 'mint a payment');
-
-    zcf.initPublicAPI(
-      harden({
-        // provide a way for anyone who knows the instanceHandle of
-        // the contract to make their own invite.
-        makeInvite,
-        // make the token issuer public. Note that only the mint can
-        // make new digital assets. The issuer is ok to make public.
-        getTokenIssuer: () => issuer,
-      }),
-    );
-
-    // return an invite to the creator of the contract instance
-    // through Zoe
-    return makeInvite();
-  });
+const start = async zcf => {
+  // Create the internal token mint for a fungible digital asset. Note
+  // that 'Tokens' is both the keyword and the allegedName.
+  const zcfMint = await zcf.makeZCFMint('Tokens');
+  // AWAIT
+
+  // Now ZCF has saved the issuer, brand, and local amountMath so that they
+  // can be accessed synchronously.
+  const { amountMath, issuer } = zcfMint.getIssuerRecord();
+
+  const mintPayment = (extent = 1000) => seat => {
```

Comment:
Clearer to move the default value onto `creatorFacet.makeInvitation` below, and just have this appear as a mandatory parameter. After all `mintPayment` is only used in that one place.

As is, the parameter to `creatorFacet.makeInvitation` looks mandatory, which is misleading.